Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,252 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+ | Antigua and Barbuda              |
+ | Bahamas                          |
+ | Belize                           |
+ | Bermuda                          |
+ | Barbados                         |
+ | Canada                           |
+ | Costa Rica                       |
+ | Cuba                             |
+ | Cayman Islands                   |
+ | Dominica                         |
+ | Dominican Republic               |
+ | Guadeloupe                       |
+ | Grenada                          |
+ | Greenland                        |
+ | Guatemala                        |
+ | Honduras                         |
+ | Haiti                            |
+ | Jamaica                          |
+ | Saint Kitts and Nevis            |
+ | Saint Lucia                      |
+ | Mexico                           |
+ | Montserrat                       |
+ | Martinique                       |
+ | Nicaragua                        |
+ | Panama                           |
+ | Puerto Rico                      |
+ | El Salvador                      |
+ | Saint Pierre and Miquelon        |
+ | Turks and Caicos Islands         |
+ | Trinidad and Tobago              |
+ | United States                    |
+ | Saint Vincent and the Grenadines |
+ | Virgin Islands, British          |
+ | Virgin Islands, U.S.             |
+ +----------------------------------+
+
+
+4.What's the top 10 countries by SurfaceArea ?
+
+ SELECT Name, SurfaceArea FROM country ORDER BY SurfaceArea DESC LIMIT 10;
+ +--------------------+-------------+
+ | Name               | SurfaceArea |
+ +--------------------+-------------+
+ | Russian Federation | 17075400.00 |
+ | Antarctica         | 13120000.00 |
+ | Canada             |  9970610.00 |
+ | China              |  9572900.00 |
+ | United States      |  9363520.00 |
+ | Brazil             |  8547403.00 |
+ | Australia          |  7741220.00 |
+ | India              |  3287263.00 |
+ | Argentina          |  2780400.00 |
+ | Kazakstan          |  2724900.00 |
+ +--------------------+-------------+
+
+
+
+
+5.What's the top 10 most populated cities?
+  
+SELECT Name FROM city ORDER BY Population DESC LIMIT 10 ;
++------------------+------------+
+| CityName         | Population |
++------------------+------------+
+| Mumbai (Bombay)  |   10500000 |
+| Seoul            |    9981619 |
+| S?o Paulo        |    9968485 |
+| Shanghai         |    9696300 |
+| Jakarta          |    9604900 |
+| Karachi          |    9269265 |
+| Istanbul         |    8787958 |
+| Ciudad de México |    8591309 |
+| Moscow           |    8389200 |
+| New York         |    8008278 |
++------------------+------------+
+
+
+
+
+6.Select the top 3 countries by population that their names start with the letter ‘P’?
+
+ SELECT Name FROM country WHERE Name LIKE 'P%' ORDER BY Population DESC LIMIT 3;
+ +-------------+
+ | Name        |
+ +-------------+
+ | Pakistan    |
+ | Philippines |
+ | Poland      |
+ +-------------+
+
+
+
+
+7.What is the population (in) Asia? 
```

Comment:
Yeah, this is a bit ambiguous - the way you did it below is what we were looking for.